project/profiles/capacity4more/modules/c4m/og/c4m_og/plugins/entityreference/selection/C4MOgSelectionHandler.class.php

```diff
@@ -87,8 +87,15 @@ public function buildEntityFieldQuery(
     }
 
     $user_groups = og_get_groups_by_user(NULL, $group_type);
-    $user_groups = $user_groups ? $user_groups : array();
+    $user_groups = $user_groups ?: array();
```

Comment:
@ybaras 
you can revert the change of this line since it does the same thing but it is less readable ...